e3sm_to_cmip/util.py

```diff
@@ -73,227 +113,240 @@ def setup_cmor(var_name, table_path, table_name, user_input_path):
-                "CMIP6 Table": handler['table'],
-                "CMIP6 Units": handler['units'],
-                "E3SM Variables":  ', '.join(handler['raw_variables'])
+                "CMIP6 Name": handler["name"],
+                "CMIP6 Table": handler["table"],
+                "CMIP6 Units": handler["units"],
+                "E3SM Variables": ", ".join(handler["raw_variables"]),
             }
-            if handler.get('unit_conversion'):
-                msg["Unit conversion"] = handler['unit_conversion']
-            if handler.get('levels'):
-                msg["Levels"] = handler['levels']
+            if handler.get("unit_conversion"):
+                msg["Unit conversion"] = handler["unit_conversion"]
+            if handler.get("levels"):
+                msg["Levels"] = handler["levels"]
             messages.append(msg)
-    
+
     # if the user asked if the variable is included in the table
     # but didnt ask about the files in the inpath
     elif freq and tables and not inpath:
         for handler in handlers:
-            table_info = get_table_info(tables, handler['table'])
-            if handler['name'] not in table_info['variable_entry']:
+            table_info = _get_table_info(tables, handler["table"])
+            if handler["name"] not in table_info["variable_entry"]:
                 msg = f"Variable {handler['name']} is not included in the table {handler['table']}"
                 print_message(msg, status="error")
                 continue
             else:
                 msg = {
-                    "CMIP6 Name": handler['name'],
-                    "CMIP6 Table": handler['table'],
-                    "CMIP6 Units": handler['units'],
-                    "E3SM Variables":  ', '.join(handler['raw_variables'])
+                    "CMIP6 Name": handler["name"],
+                    "CMIP6 Table": handler["table"],
+                    "CMIP6 Units": handler["units"],
+                    "E3SM Variables": ", ".join(handler["raw_variables"]),
                 }
-                if handler.get('unit_conversion'):
-                    msg["Unit conversion"] = handler['unit_conversion']
-                if handler.get('levels'):
-                    msg["Levels"] = handler['levels']
+                if handler.get("unit_conversion"):
+                    msg["Unit conversion"] = handler["unit_conversion"]
+                if handler.get("levels"):
+                    msg["Levels"] = handler["levels"]
                 messages.append(msg)
-    
+
     elif freq and tables and inpath:
-        file_path = next(Path(inpath).glob('*.nc'))
-        
-        resource_path, _ = os.path.split(os.path.abspath(resources.__file__))
-        defaults_path = os.path.join(
-            resource_path,
-            'default_handler_info.yaml')
-        
-        with open(defaults_path, 'r') as infile:
-            default_info = yaml.load(infile, Loader=yaml.SafeLoader)
-        
```

Comment:
Loading default handlers is no longer required for the high frequency check below.